server/apps/archive/archive_composite.py

```diff
@@ -134,8 +134,11 @@ def update_link(self, package_id, assoc, delete=False):
         if not delete:
             two_way_links.append({'package': package_id})
 
+        # do update without changing etag/last_modified
         updates = self.get_item_update_data(item, two_way_links, delete)
-        get_resource_service(endpoint).patch(item_id, updates)
+        updates['_etag'] = item['_etag']
```

Comment:
I think that a good approach is that the saved etag to have the same value as the one calculated on values, otherwise we will have some issues like those that we have now for user preferences that are failing on save because of this matter.